docs/narr/scaffolding.rst

```diff
@@ -5,7 +5,7 @@ Creating Pyramid Scaffolds
 
 .. deprecated:: 1.8
 
-    Scaffolds and the ``pcreate`` script used to generate :app:`Pyramid` projects from scaffolds have been deprecated. Use :ref:`cookiecutters` instead.
+    Scaffolds and the ``pcreate`` script used to generate :app:`Pyramid` projects from scaffolds have been deprecated. Use :ref:`cookiecutter` instead.
```

Comment:
Let's not change this reference because Sphinx will convert this to the heading to which that reference is bound, namely the text "Pyramid cookiecutters" in `narr/cookiecutters.rst`.

Additionally, we should keep the heading plural. We want to continue to mention previous cookiecutters for the 1.9-branch until we no longer support that branch. In the 1.10-branch, we can deprecate the older separate CCs, and announce their replacement by the single unified cookiecutter. This will also allow the build to get past this failure.